Add a contributing section to the documentation (#497)
containous/maesh: docs/content/contributing/submitting-issues.md

# Submitting Issues

We use the [GitHub issue tracker](https://github.com/containous/maesh/issues) to keep track of Maesh issues. 

The process of sorting and checking the issues requires a lot of work. To save us some time and get quicker feedback, 
be sure to follow the guidelines below.

!!! Important "Getting Help"
    The issue tracker is not a general support forum, but a place to report bugs and asks for new features. For end-user 
    related support questions, use the [Maesh community forum](https://community.containo.us/c/maesh/15).

## Issue Title

The title must be short and descriptive. (~60 characters)

## Description

Follow the [issue template](https://github.com/containous/maesh/blob/master/.github/ISSUE_TEMPLATE/) as much as possible.

> Explain us in which conditions you encountered the issue, what is your context.

Reviewer: ```suggestion
Explain in which context you encountered the issue.
```

Remain clear and concise.

Take time to polish the format of your message so we'll enjoy reading it and working on it. Help the readers focus on 
what matters, and help them understand the structure of your message (see the [Github Markdown Syntax](https://help.github.com/articles/github-flavored-markdown)).

## Feature Request

Remember, when asking for new features, they must be useful to the majority (and not only useful in edge case scenarios, or hack-like setups).

Do your best to explain what you're looking for, and why it would improve Maesh for everyone. 

## International English

> Every Maesh maintainer/user is not a native English speaker, so if you sometimes feel that some messages sound rude, 

Reviewer: ```suggestion
Maesh maintainers/users are not all native English speakers, so if you sometimes feel that some messages sound rude, 
```
